lms/static/scripts/frontend_apps/config.js

```diff
@@ -76,6 +76,10 @@ import { createContext } from 'preact';
  *   @prop {boolean} vitalSource.enabled
  */
 
+/**
+ * @typedef {'blackboard_missing_integration'} OAuthErrorCode
```

Comment:
No idea if this is useful at all, maybe TS will provide exhaustiveness check if used in a case statement?

Reply:
I don't think TS currently does exhaustiveness checks, but it will complain if you try to test it against a value that it can't have (for example if you made a typo and tried to compare an `OAuthErrorCode` value to `blackbard_missing_integration`). It's also just useful documentation for readers.